docs/manual/layout/templates/twig/namespace.en.md

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Namespaces'
+description: 'Namespaces for Twig templates'
+aliases:
+    - /en/layout/templates/twig/namespace/
+weight: 80
+---
+
+
+The full template name inside Twig contains a "namespace". And a namespace looks like `@ContaoCore/Image/Studio/figure.html.twig`.
+
+Our Contao templates are all inside the `@Contao` namespace, so that's why for instance the text content element's 
+"fully qualified template name" is `@Contao/content_element/text.html.twig` and why you would extend from it with 
+`{% extends "@Contao/content_element/text.html.twig" %}`.
+
+The `@Contao` namespace is a "managed namespace" and has a special feature in contrast to standard Twig: The same template can be 
+extended from various sources. This way an extension can, for instance, add a new feature to a core template while you can still adjust it 
+in your application.
+
+{{% notice tip %}}
+You can run `contao-console debug:contao-twig` on the console to get a list of all registered namespaces. 
+{{% /notice %}}
```

Comment:
IMHO this will unneccessarily confuse people. I'D remove it. :slightly_smiling_face: 

```suggestion

{{% notice tip %}}
You can run `contao-console debug:contao-twig` on the console to get a list of all registered namespaces. 
{{% /notice %}}
```